app/assets/stylesheets/custom.scss

```diff
@@ -155,11 +156,66 @@ th {
   text-align: center;
 }
 
+// about page
+.about {
+  background-color: #e3e3e3;
+  border: 2px solid #6699ff;
+  border-radius: 20px;
+  margin-bottom: 2px;
+  margin-left: auto;
+  margin-right: auto;
+  min-height: 20px;
+  padding: 50px;
+  width: 600px;
+}
+
+// footer
+footer {
+  background-color: #6bc5f2;
+  border-top: 1px solid #eaeaea;
+  bottom: 0;
+  height: 60px;
+  position: relative;
+  width: 100%;
+}
+
+footer a {
+  color: #894355;
+  font-size: 1.3em;
+  margin-right: 15px;
+}
+
+footer a:hover {
+  color: #222;
+}
+
+footer small {
+  float: left;
+  font-size: 0.9em;
+}
+
+footer ul {
+  float: right;
+  text-decoration: underline;
+}
+
+footer ul li {
```

Comment:
Merge rule `footer ul li` with rule on line 173<br>Selector should have depth of applicability no greater than 2, but was 3